tests/nested/manual/remodel-uc20-to-uc22/task.yaml

```diff
@@ -21,20 +30,23 @@ environment:
   # vault
   SERIAL: 1234-snapd-remodel-minsize
   GADGET_NAME: test-snapd-remodel-pc
-  NEW_GADGET_NAME: test-snapd-remodel-pc-min-size
+  # use new gadget with shim signed by snakeoil keys so encrypted case works
+  NEW_GADGET_NAME: test-snapd-remodel-pc-snakeoil
```

Comment:
I'm not sure I remember where is this new gadget defined/coming from?

Reply:
This is a gadget packed from tests/lib/snaps/test-snapd-remodel-pc-min-size-22/, but with shim re-signed. I've added a comment to clarify.